command.go

```diff
@@ -22,6 +26,92 @@ type ExecCommand struct {
 	description string
 }
 
+// ConfigCommand is a Command that sets some saved state
+type ConfigCommand struct {
+	Desc    string
+	Updater func(c Config) (Config, error)
+}
+
+// ToggleDryRunCommand is a special command that toggles dry run state
+type ToggleDryRunCommand struct{}
+
+// Config is the state of the build bot.
+// DryRun will print out what it plans to do without doing it
+// Paused will prevent any builds in the future from running
+type Config struct {
+	DryRun bool
+	Paused bool
+}
+
+func getConfigPath() (string, error) {
+	u, err := user.Current()
```

Comment:
I know golang wants to you use short variable names.. but I think they are wrong...

I would use `currentUser` instead of `u` here